flycheck.el

```diff
@@ -8164,6 +8164,22 @@ See URL `https://eslint.org' for more information about ESLint."
           (let-alist (caar (flycheck-parse-json output))
             .messages)))
 
+(defun flycheck-eslint--find-working-directory (_checker)
+  "Look for a working directory to run CHECKER in.
```

Comment:
You forgot to add a newline after the first sentence.  After that, all good!

Reply:
done.